code/modules/mob/living/simple_animal/hostile/human/mummy.dm

```diff
@@ -0,0 +1,221 @@
+//Mummies
+//Very slow mobs with a lot of health
+//On death, create a cloud of toxins
+
+/mob/living/simple_animal/hostile/humanoid/mummy
+	name = "mummy"
+	desc = "A mummified corpse of a common man, most likely a slave who was buried with their master."
+
+	icon = 'icons/mob/hostile_humanoid.dmi'
+	icon_state = "mummy"
+
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
+
+	health = 70
+	maxHealth = 70
+
+	move_to_delay = 20
+	speed = 4
+
+	melee_damage_lower = 3
+	melee_damage_upper = 6
+	stat_attack = UNCONSCIOUS //Attack unconscious dudes too
+
+	faction = "mummy"
+	corpse = /obj/effect/landmark/corpse/mummy
+
+/obj/effect/landmark/corpse/mummy
+	name = "mummy"
+
+	husk = 1
+	corpseuniform = /obj/item/clothing/under/mummy_rags
+	corpsehelmet = /obj/item/clothing/head/mummy_rags
+
+/mob/living/simple_animal/hostile/humanoid/mummy/Die()
+	if(!isturf(loc))
+		return
+
+	visible_message("<span class='danger'>\The [src] emits a cloud of miasma before being put to peace.</span>")
+	var/datum/effect/effect/system/smoke_spread/chem/rot/S = new /datum/effect/effect/system/smoke_spread/chem/rot
+	S.attach(loc)
+	S.set_up(src, 10, 0, loc)
+	spawn(0)
+		S.start()
+
+	..()
+
+/datum/effect/effect/system/smoke_spread/chem/rot
+	set_up(var/mob/M, n = 5, c = 0, loca, direct)
+		if(n > 20)
+			n = 20
+		number = n
+		cardinals = c
+
+		chemholder.reagents.add_reagent("toxin", 50)
+		chemholder.reagents.add_reagent("cyanide", 20)
```

Comment:
You do know cyanide at that dosage is extremely deadly, right? If I were you, I'd replace it with sleep toxin or even 5-8u of chloral hydrate.